spec/lib/declare_schema/api_spec.rb

```diff
@@ -41,10 +41,12 @@ class Advert < #{active_record_base_class}
 
       if ActiveSupport::VERSION::MAJOR == 5
         # TODO: get this to work on Travis for Rails 6
+        # TODO: uncomment since we're not on Travis any more? -Colin
```

Comment:
Seems like we should try sometime.

Reply:
Wanna try this just before merge and if it works merge if it doesn't revert and create a new ticket?

Reply:
Yeah, I can try.

Reply:
It didn't work. And I'm actually not sure why that line is needed there for Rails 5?

So for now I deleted the comments here, and also the one below about rich types.